gamestonk_terminal/portfolio_optimization/port_optimization_controller.py

```diff
@@ -0,0 +1,277 @@
+"""Portfolio Optimization Controller Module"""
+__docformat__ = "numpy"
+
+
+import argparse
+from typing import List
+import matplotlib.pyplot as plt
+import pandas as pd
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair, parse_known_args_and_warn
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.portfolio_optimization import port_opt_api as po_api
+from gamestonk_terminal.comparison_analysis import ca_controller
+from gamestonk_terminal.screener import screener_controller
+
+
+class PortfolioOptimization:
+
+    CHOICES = [
+        "help",
+        "q",
+        "quit",
+        "ca",
+        "scr",
+        "select",
+        "add",
+        "equal_weight",
+        "mkt_cap",
+        "div_yield",
+        "max_sharpe",
+        "min_vol",
+        "eff_risk",
+        "eff_ret",
+        "show_ef",
+        "yolo",
+    ]
+
+    def __init__(self, tickers: List[str] = None):
+        """
+        Construct Portfolio Optimization
+        """
+
+        self.po_parser = argparse.ArgumentParser(add_help=False, prog="po")
+        self.po_parser.add_argument("cmd", choices=self.CHOICES)
+        self.tickers = set(tickers)
+        # These will allow the ca menu to be re-access
+        self.ca_ticker = None
+        self.ca_similar = None
+
+    @staticmethod
+    def print_help(tickers: List[str]):
+        """Print help"""
+        print("\nPortfolio Optimization:")
+        print("   help          show this menu again")
+        print("   q             quit this menu, and shows back to main menu")
+        print("   quit          quit to abandon program")
+        print("   > ca          comparison analysis menu")
+        print("   > scr         screener menu")
+        print(f"\nCurrent Tickers: {('None', ', '.join(tickers))[bool(tickers)]}")
+        print("")
+        print("   add          add ticker to optimize")
+        print("   select       overwrite current tickers with new tickers")
+        print("")
+        print("Optimization:")
+        print("")
+        print("   Property weighted:")
```

Comment:
You can remove this, and go with your first idea.

Let all the command options with only 3 spaces, e.g.
```
print("   mkt_cap        marketcap weighted portfolio")
print("   div_yield       dividend weighted portfolio")
...
```

Reply:
so no property weighted or MVO?  Just list all options?

Reply:
Exactly. As you mentioned the first time